front/lib/development.ts

```diff
@@ -17,6 +17,10 @@ export function canForceUserRole(owner: LightWorkspaceType) {
   return isDevelopment() || isADustProdWorkspace(owner);
 }
 
+export function canEnableJIT(owner: LightWorkspaceType) {
```

Comment:
Can we move this function to jit_actions so that we have no export to avoid leaving it here once we enable JIT globally :pray:

Reply:
sure